### DISCUSSION TITLE
Structure a wokrspaced repository to deploy to multiple subdomains

### DISCUSSION THREAD

**Discussion by fabien-h** [vercel/vercel#4647] [Help]
I have a repository structure that looks like this :

```
src
└─ package
   └─ api (to deploy to api.ndd.tld)
   └─ app (to deploy to app.ndd.tld)
   └─ blog (to deploy to blog.ndd.tld)
   └─ components
   └─ front (to deploy to ndd.tld and www.ndd.tld)
   └─ mobile (react native applications)
   └─ storybook
   └─ types
```

I'm not sure to understand how this works with vercel. Should I have a single vercel.json or one per package ?

How do I also deploy to `api.dev.ndd.tld` and `api.staging.ndd.tld` ?

**Comment by paulogdm** [comment]
"Monorepo" support is what you are looking for. It is going to arrive soon. Please send us a message with your email + link to this discussion on https://vercel.com/support/request and let us know.

**Reply by fabien-h** [reply]
Thank you for your time. Yes, it's a monorepo multi target. You will go for lerna or a simple yarn workspace ?

Is it soon as "next week, it's already in the canary channel" or soon as "early 2021" ? :)

Not sure to understand the support request. Is it to tell the support team that one more dude would like to have it ? Or to see a preview ?

**Reply by paulogdm** [reply]
We can add you to the list of interested customers to beta-test the feature.

We plan to launch it it this year since it is a high priority item.

**Reply by mcsdevv** [reply]
Hey @fabien-h, this is now available! You can read more about monorepo support here: https://vercel.com/blog/monorepo

**Reply by fabien-h** [reply]
@mcsdevv  Awesome !

(but I get a 404 on your link)

[EDIT] : https://vercel.com/blog/monorepos , you forgot the final s :)
